x-pack/plugins/security_solution/public/common/components/event_details/cti_details/threat_summary_view.tsx

```diff
@@ -7,171 +7,335 @@
+                }
+              />
+            ))}
+          </EuiPanel>
+        </>
+      )}
+    </>
+  );
 };
 
-const columns: Array<EuiBasicTableColumn<ThreatSummaryItem>> = [
-  {
-    field: 'title',
-    truncateText: false,
-    render: EnrichmentTitle,
-    width: '220px',
-    name: '',
-  },
-  {
-    className: 'flyoutOverviewDescription',
-    field: 'description',
-    truncateText: false,
-    render: EnrichmentDescription,
-    name: '',
-  },
-];
+const HostRiskDataBlock: React.FC<{
+  hostRisk: HostRisk;
+}> = ({ hostRisk }) => (
+  <>
+    <EuiPanel hasBorder paddingSize="s" grow={false}>
+      <ThreatSummaryPanelHeader
+        title={i18n.HOST_RISK_DATA_TITLE}
+        toolTipContent={
+          <FormattedMessage
+            id="xpack.securitySolution.alertDetails.overview.hostDataTooltipContent"
+            defaultMessage="Risk classification is displayed only when available for a host. Ensure {hostsRiskScoreDocumentationLink} is enabled within your environment."
+            values={{
+              hostsRiskScoreDocumentationLink: (
+                <EuiLink href={RISKY_HOSTS_DOC_LINK} target="_blank">
+                  <FormattedMessage
+                    id="xpack.securitySolution.alertDetails.overview.hostsRiskScoreLink"
+                    defaultMessage="Host Risk Score"
+                  />
+                </EuiLink>
+              ),
+            }}
+          />
+        }
+      />
+
+      {hostRisk.loading && <EuiLoadingSpinner />}
+
+      {!hostRisk.loading && (!hostRisk.isModuleEnabled || !hostRisk.result) && (
+        <EuiText color="subdued" size="xs">
+          {i18n.NO_HOST_RISK_DATA_DESCRIPTION}
+        </EuiText>
+      )}
+
+      {hostRisk.isModuleEnabled && hostRisk.result && hostRisk.result.length > 0 && (
+        <>
+          <EnrichedDataRow field={'host.risk.keyword'} value={hostRisk.result[0].risk} />
+        </>
+      )}
+    </EuiPanel>
+  </>
+);
 
 const ThreatSummaryViewComponent: React.FC<{
   browserFields: BrowserFields;
   data: TimelineEventsDetailsItem[];
   enrichments: CtiEnrichment[];
   eventId: string;
   timelineId: string;
-}> = ({ browserFields, data, enrichments, eventId, timelineId }) => (
-  <Indent>
-    <StyledEuiInMemoryTable
-      columns={columns}
-      compressed
-      data-test-subj="threat-summary-view"
-      items={buildThreatSummaryItems(browserFields, data, enrichments, timelineId, eventId)}
-    />
-  </Indent>
-);
+  hostRisk: HostRisk | null;
+  isDraggable?: boolean;
+}> = ({ browserFields, data, enrichments, eventId, timelineId, hostRisk, isDraggable }) => {
+  if (!hostRisk && enrichments.length === 0) {
+    return null;
+  }
+
+  return (
+    <>
+      <EuiHorizontalRule />
+
+      <EuiTitle size="xxxs">
+        <h5>{i18n.ENRICHED_DATA}</h5>
+      </EuiTitle>
+      <EuiSpacer size="m" />
+
+      {hostRisk && <HostRiskDataBlock hostRisk={hostRisk} />}
+
+      {hostRisk && enrichments.length > 0 && <EuiSpacer size="m" />}
```

Comment:
we could probably put the spacer into the HostRiskDataBlock

Reply:
But then we would need to pass down `enrichments ` as props to `HostRiskDataBlock`. And it doesn't make much sense for it to know about enrichments. We could get rid of `EuiSpacer` if we create a flex-box wrapper and ensure that the panels are evenly spaced with `row-gap`.